.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Custom issue template
+about: https://circleci.atlassian.net/browse/AIT-8996
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Description
+
+> Describe what's the issue is
```

Comment:
Typo

```suggestion
> Describe the issue in detail
```

Reply:
This is coming from this request: https://circleci.atlassian.net/browse/AIT-8996

Reply:
I don't have access. Can you grant me access to this issue?

----

Regardless, this phrase has 2 verbs. It's grammatically incorrect. Expanding the contraction gives:

```
> Describe what is the issue is
```

"what's" being a colloquial term, I propose we use a more formal sentence, and eliminate grammatical errors, unless imperative.